src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -271,17 +278,42 @@ object ZincRunner extends WorkerMain[Namespace] {
       }
 
     // create analyses
-    analysisStore.set(AnalysisContents.create(compileResult.analysis, compileResult.setup))
+    val path = analysisStoreFile.getCanonicalPath()
+    val analysisStoreText = AnalysisUtil.getAnalysisStore(
+      new File(path.substring(0, path.length() - 3) + ".text.gz"),
+      true,
+      usePersistence,
+    )
+    // Filter out libraryClassNames from the analysis because it is non-deterministic.
+    // Can stop doing this once the bug in Zinc is fixed. Check the comment on FilteredRelations
+    // for more info.
```

Comment:
is there a bug filed against zinc for this?

Reply:
Not yet, but I intend to fix it when I have a moment. Hopefully sometime in the next couple months.